lib/jsBundler.js

```diff
@@ -0,0 +1,48 @@
+// js bundler
+
+require('colors');
+
+var browserify = require('browserify'),
+    fs = require('fs'),
+    fse = require('fs-extra');
```

Comment:
I hesitant at needing to require `fs` here, as `fse` abstracts all of `fs` into it. The only problem I can see is `fse`, could be slow to get to `fs` updates